reporting/api/src/main/java/org/jboss/windup/reporting/model/JarDependenciesReportModel.java

```diff
@@ -0,0 +1,33 @@
+package org.jboss.windup.reporting.model;
+
+import org.jboss.windup.graph.model.ProjectModel;
+
+import com.tinkerpop.blueprints.Direction;
+import com.tinkerpop.frames.Incidence;
+import com.tinkerpop.frames.modules.typedgraph.TypeValue;
+
+/**
+ * This represents Jar Dependencies report model
+ * 
+ * @author mnovotny
+ *
+ */
+@TypeValue(JarDependenciesReportModel.TYPE)
+public interface JarDependenciesReportModel extends ApplicationReportModel
+{
+    String TYPE = "jarDependenciesReport";
+
+    String JAR_REPORT_TO_PROJECT_MODEL = "jarDepsToProjectModel"; 
+
+    /**
+     * Contains all {@link ProjectModel}s that contain this file
+     */
+    @Incidence(label = JAR_REPORT_TO_PROJECT_MODEL, direction = Direction.OUT)
```

Comment:
honestly I don't get what would be useful in this case. I thought that only thing I need is to get ProjectModel which holds the information about an archive and all duplicated archives which are already vertexes pointing to a Project model.

That is why I joined new Jar Dependency Report model to project model and archive model, where is the information about path.